src/test/java/com/salesforce/dynamodbv2/mt/mappers/MTAmazonDynamoDBBySharedTableTest.java

```diff
@@ -43,7 +43,9 @@
 import static com.amazonaws.services.dynamodbv2.model.ScalarAttributeType.S;
 import static java.util.UUID.randomUUID;
 
-/**m
+/**
+ * m
```

Comment:
this comment seems... minimalistic?

Reply:
This was a pre-existing mistake.  I saw it when I was reviewing.  I was tempted to just fix it but then reminded myself that this was just a ctrl-alt-l effort first.

Reply:
all righty